.github/workflows/ci.yml

```diff
@@ -9,20 +9,28 @@ on:
 
 jobs:
   tests:
-    name: "Python ${{ matrix.python-version }}"
+    name: "Python ${{ matrix.python }}"
     runs-on: "ubuntu-latest"
     env:
-      USING_COVERAGE: "3.7"
-
+      USING_COVERAGE: "3.9"
     strategy:
       matrix:
-        python-version: ["3.7"]
+        include:
+        - name: py37
+          python: 3.7
+        - name: py38
+          python: 3.8
+        - name: py39
+          python: 3.9
+          coverage: "--cov"
```

Comment:
why `--cov` wouldn't `true` be enough and more clear?

Reply:
Good point. Changed it to `true`